index.html

```diff
@@ -2933,10 +2931,10 @@ <h3>
 
           <p>
   If <code>returnFullDocuments</code> was set to false, a successful query will result in a standard HTTP 200 response with a list
-  of <a>EncryptedDocument</a> identifiers that contain the value:
+  of <a>EncryptedDocument</a> identifiers that contain the attribute key:
```

Comment:
Shouldn't this be attribute value?

Reply:
These are the "has" query examples that only use an attribute key - but regardless, I changed it to say "...that match the query" to be more consistent with the previous examples - so you just have to check the example above it to see the query type